.github/workflows/surge-preview-build.yaml

```
@@ -5,6 +5,10 @@ on:
     types: [opened, synchronize, reopened]
     paths:
       - 'web/**'
+    paths-ignore:
```

actionlint (1.7.9)

both "paths" and "paths-ignore" filters cannot be used for the same event "pull_request". note: use '!' to negate patterns

(events)